targets/ar71xx-generic

```diff
@@ -184,6 +184,18 @@ device tp-link-archer-c7-v2 archer-c7-v2
 packages $ATH10K_PACKAGES
 factory -squashfs-factory${GLUON_REGION:+-${GLUON_REGION}} .bin
 
+if [ "$BROKEN" ]; then
+device tp-link-archer-c58-v1 archer-c58-v1
+fi
+
+if [ "$BROKEN" ] || [ "$GLUON_ATH10K_MESH" = "11s" ]; then
```

Comment:
this line has to be adjusted to use the new GLUON_WLAN_MESH variable instead ( #1298 )